native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -36,14 +36,16 @@ const styles = StyleSheet.create({
   horizontalNav: {
     borderColor: 'transparent',
     borderBottomColor: colorGray100,
+    flexDirection: 'column',
     borderWidth: borderSizeSm,
   },
+  flexGrow: {
```

Comment:
As I remember the reason for this was a bit intricate, could we incldue a comment explaining it so we don't end up confused about why this is here

Reply:
Fair point, will do